<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserWidget</class>
 <widget class="QWidget" name="LaserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>激光数据(上)</string>
   </property>
   <widget class="QPushButton" name="btnLaserDataGet">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>激光数据获取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLaserDataClear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>激光数据清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLaserDataTest">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>激光数据测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>激光数据(下)</string>
   </property>
   <widget class="QPushButton" name="btnLaserDataClear_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>激光数据清空</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLaserDataGet_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>激光数据获取</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLaserDataTest_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>激光数据测试</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>数据刷新周期</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>392</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QLineEdit" name="leDataRefCycle"/>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QPushButton" name="btnRefEnsure">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QPushButton" name="btnRefStop">
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>拉杆激光</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>底部激光</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>结果检测</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>95</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="laserOneBtn">
       <property name="text">
        <string>1_laser</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="laserTwoBtn">
       <property name="text">
        <string>2_laser</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="laserThreeBtn">
       <property name="text">
        <string>3_laser</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="laserFourBtn">
       <property name="text">
        <string>4_laser</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>95</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="laserCrossOneBtn">
       <property name="text">
        <string>1_2laser</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="laserCrossTwoBtn">
       <property name="text">
        <string>1_4laser</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>坐标显示</string>
   </property>
   <widget class="QListWidget" name="position_listWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnclearlist">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>570</y>
     <width>275</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icons/big_starticon.png</normaloff>:/icons/big_starticon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resultBtn">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/images/smile.png</normaloff>
        <normalon>:/images/images/cry.png</normalon>:/images/images/smile.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
